.github/workflows/2-step.yml

```
@@ -0,0 +1,147 @@
+name: Step 2
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/python-tests.yml
```

`python-tests.yml` is never created 

```suggestion
      - .github/workflows/python-package.yml
```

---

This is really strange. I swear I already fixed this. 🧐